src/modules/HomePage/hooks/useSortByLatestMessage.ts

```diff
@@ -24,7 +24,7 @@ export default function useSortedPostIdsByLatestMessage(
     const hasAddedIds = new Set()
     const sortedIds: string[] = []
     posts.forEach((post) => {
-      const id = (post as CommentData)?.struct.rootPostId
+      const id = (post as unknown as CommentData)?.struct.rootPostId
```

Comment:
I don't use it because it will increase the first load js for home page into 526kb from 164kb 😅

Reply:
Wow, it is huge problem of our SDK, it happens because SDK includes @polkadot/api, right?

Reply:
no, actually I checked and I suspect its because it includes summarizeMd in that same file (I don't know if it includes the md things only or all other things)
![image](https://user-images.githubusercontent.com/53143942/232787634-cde66ec7-56d4-4c1c-920c-8a919fa83934.png)

but I think if you now import @subsocial/utils importing only 1 function, you also import everything inside of it, because we export everything from index.ts, but this may not be right, need to check it further